src/api/spec/features/webui/packages_spec.rb

```diff
@@ -96,4 +97,39 @@
     expect(page).to have_text("In state review")
     expect(page).to have_text("Set the devel project to package #{third_project.name} / develpackage for package #{user.home_project} / develpackage")
   end
+
+  context "triggering package rebuild" do
+    let(:repository) { create(:repository, architectures: ["x86_64"]) }
+    let(:rebuild_url) {
+      "http://localhost:3200/build/#{user.home_project.name}?cmd=rebuild&arch=x86_64&package=#{package.name}&repository=#{repository.name}"
+    }
+    let(:fake_buildresult) {
```

Comment:
I would prefer to return a `Xmlhash.parse("<resultlits ....>")` instead of a string, so is like moving the call of line 128 to here.